raptorWeb/staffapps/forms.py

```diff
@@ -0,0 +1,156 @@
+from django import forms
+from django.core import validators
+import logging
+
+from staffapps.models import ModeratorApplication, AdminApplication
+from staffapps.util import form_labels
+
+LOGGER = logging.Logger("form_validator_logger")
+
+def check_for_hash(value):
+    """
+    Ensure Discord name value contains a '#' symbol
+    """
+    if value.find("#") < 0:
+
+        raise forms.ValidationError("Format your Discord Username as indicated in the help text.")
+
+def validate_age(value):
+    """
+    Ensure age value is at least 10
+    """
+    if value < 10:
+
+        raise forms.ValidationError("You must be at least ten years old to apply for staff")
+
+def validate_admin_age(value):
+    """
+    Ensure age value is at least 18
+    """
+    if value < 18:
+
+        raise forms.ValidationError("You must be at least 18 years old to apply directly for Admin. One can still be promoted from a lower rank.")
+
+def verify_minecraft_username(clean_data):
```

Comment:
Could abstract `verify_discord_username` and `verify_minecraft_username` into a simple comparison function, taking the different cleaned data elements as arguments

Reply:
This would just take more code than leaving as is in the current implementation